crates/bevy_math/src/primitives/dim2.rs

```diff
@@ -1817,6 +1817,28 @@ impl Capsule2d {
             half_length: length / 2.0,
         }
     }
+
+    /// Get the part connecting the semicircular ends of the capsule as a [`Rectangle`]
+    #[inline(always)]
```

Comment:
```suggestion
    #[inline]
```
Please try to avoid `inline(always)` unless it's shown that the function will not inline with just the base annotation.